PowerEditor/src/WinControls/TabBar/ControlsTab.h

```diff
@@ -38,7 +38,7 @@ struct DlgInfo
     generic_string _name;
 	generic_string _internalName;
 
-	DlgInfo(Window *dlg, TCHAR *name, TCHAR *internalName = NULL): _dlg(dlg), _name(name), _internalName(internalName?internalName:TEXT("")) {};

```

Comment:
NON TRIVIAL CHANGE  - changed default parameter from `NULL` to `""` so there is no need for this conditional check. Checking through all uses of this constructor, no one ever passes `NULL` to it so current behaviour stays unchanged.